### PR TITLE
Add images and layout to prominence page

### PR DESCRIPTION
**Pull request by KenjiCrosland**

### CODE REVIEW

File: docs/.vuepress/components/TwoColumnGrid.vue [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<template>
```
Comment (KenjiCrosland) [inline docs/.vuepress/components/TwoColumnGrid.vue:1]: This is essentially a stripped down `DoDont` component

File: docs/.vuepress/components/TwoColumnGrid.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            <slot/>
+          </figcaption>
+        </figure>
+      </div>
+    </cdr-grid>
+  </div>
+</template>
+
+<script>
+import Vue from '$vue';
```
Comment (benjag) [inline docs/.vuepress/components/TwoColumnGrid.vue:25]: Don't think you need this import, it brings in the full build of vue ... runtime compiler and alll

Comment (KenjiCrosland) [reply docs/.vuepress/components/TwoColumnGrid.vue]: Yup removed it and everything works. Removed the import from the `<DoDont>` component I used as a template as well.

File: docs/.vuepress/components/TwoColumnGrid.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          </figcaption>
+        </figure>
+      </div>
+    </cdr-grid>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "TwoColumnGrid",
```
Comment (benjag) [inline docs/.vuepress/components/TwoColumnGrid.vue:26]: I think the component name should a little more specific to what this component is actually doing, there's probably going to be a couple different components that render content in a two-col grid

**Pull request closed without merging**